packages/slice-machine/lib/models/common/widgets/ContentRelationship/index.ts

```diff
@@ -26,15 +27,15 @@ const Meta = {
   description: "Define content relations & internal links",
 };
 
-const configSchema = yup.object().shape({
+const contentRelationShipConfigSchema = linkConfigSchema.shape({
   label: yup.string().max(35, 'String is too long. Max: 35'),
```

Comment:
I think this condition should be in the original link schema or removed.
Is it reflected in the UI ? Like preventing people from creating big labels ?

Anyway, this schema should be clear like the media one and add only what's specific about the content relationship :)